src/Controller/MemberController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+
+namespace App\Controller;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class MemberController extends AbstractController
+{
+    /**
+     * @Route("/espace-client/{id}", name="customer_space")
+     * @param User $user
+     * @return  Response
+     */
+    public function show(User $user): Response
+    {
+        return $this->render('user/show.html.twig', [
```

Comment:
ici il faut plutot récupérer le user actuellement connecté avec un 
$user = $this->getUser();